doc/rados/operations/balancer.rst

```diff
@@ -60,7 +60,8 @@ of the following form:
 A larger value may increase the speed of cluster balancing / convergence
 at the potential cost of greater impact on client operations.
 
-There is a separate setting for how uniform the distribution of PGs must be for
+There is a separate setting ``upmap_max_deviation``
```

Comment:
I suggest wrapping the option name in a `confval`, there are examples in other files in this directory.